### PR TITLE
Update deploy-and-configure-the-report-message-add-in.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10938

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/step-by-step-guides/deploy-and-configure-the-report-message-add-in.md [hunk 1]
```diff
@@ -54,9 +54,8 @@ Depending on whether you are licensed for Defender for Office 365, you'll also g
 ## Configure the add-in for users
 
 1. **Login** to the Microsoft Security portal at <https://security.microsoft.com>.
-2. On the left nav, under **Email & collaboration**, select **Policies & rules**.
-3. Select **Threat policies**.
-4. Select **User reported message settings** underneath the **Others** heading.
+2. On the left nav, select **Settings** and choose **Email & collaboration**.
+3.  Select **User reported message settings**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/step-by-step-guides/deploy-and-configure-the-report-message-add-in.md]:
```suggestion
3.  Select **User reported message settings**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Pull request merged**